content/influxdb/cloud-iox/sql-reference.md

```diff
@@ -0,0 +1,614 @@
+---
+title: InfluxDB SQL reference
+description: >
+  InfluxDB SQL reference
+menu:
+  influxdb_cloud_iox:
+    name: InfluxDB SQL reference
+    parent: Query data with SQL
+weight: 190
+---
+
+InfluxDB Cloud backed by InfluxDB IOx uses the Apache Arrow DataFusion implementation of SQL.  
+
+- [Identifiers](#identifiers)  
+- [Quoting and case sensitivity](#quoting-and-case-sensitivity)    
+- [Literals](#literals)  
+- [Duration units](#duration-units)  
+- [Operators](#operators)  
+- [Keywords](#keywords)  
+- [Conditional expressions](#conditional-expressions)
+- [Statements and clauses](#statements-and-clauses)  
+- [Comments](#comments)  
+- [Functions](#functions)  
+
+## Identifiers
+
+An identifier is a token which refers to the name of an InfluxDB database object, such as a **measurement** or a column name (**time**, **tag keys**, or **field keys**).
+
+## Quoting
+
+Use double quotes on [identifiers](#identifiers) to treat them as case-sensitive.
+Use single quotes on string literals.
+
+General quoting guidelines:
+
+- Single quote RFC3339 and RFC3339-like time values.
+- Do _not_ quote Unix epoch time values (integers cast to a timestamp).
+- Double-quote mixed case, [camel case](https://en.wikipedia.org/wiki/Camel_case) or case-sensitive identifiers.
+- Double-quote identifiers that contain special characters or whitespace characters.
+
+##### Quoting examples
+
+```sql
+-- Double-quote identifiers that contain whitespace
+SELECT "water temperature", "buoy location" FROM buoy
+
+-- Double-quote measurement names with special characters
+SELECT * FROM "h2o-temperature"
+
+-- Double-quote identifiers that should be treated as case-sensitive
+SELECT "pH" FROM "Water"
+```
+
+{{% note %}}
+**Note:** We recommend always double-quoting identifiers, regardless of case-sensitivity.
+{{% /note %}}
+
+
+Unquoted identifiers **are not** case-sensitive and match any measurement, tag key, or field key with the same characters, despite case.
+For example, if you have two fields in a measurement named `ph` and `pH`, the unquoted identifier, `pH` will match both.
```

Comment:
I actually think the unquoted `pH` will only match `ph` 

I agree with the suggestion to always double quote identifiers